### PR TITLE
Nbconvert CSS bundler

### PR DESCRIPTION
**Pull request by SylvainCorlay**
This new package produces a CSS bundle for the JupyterLab nbconvert theme.

I think that it is interesting for it to be included in the core JupyterLab packages so that it is continuously updated by `lerna` as new jupyterlab packages are released.

This is using webpack to produce the standalone bundle. There are a couple of small images that are inlined in the resulting CSS.

cf. https://github.com/jupyter/nbconvert/pull/1005 for the nbconvert PR moving nbconvert from the classic notebook to the JupyterLab DOM structure.

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
cc @jasongrout.

**Comment by blink1073** [comment]
Thanks @SylvainCorlay!   Mind removing the package-lock file?

**Comment by blink1073** [comment]
M

**Comment by SylvainCorlay** [comment]
> Thanks @SylvainCorlay! Mind removing the package-lock file?

Done! Let me know if you prefer this to be 0.1.0 instead of 0.0.1.

File: packages/nbconvert-css/package.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@jupyterlab/nbconvert-css",
+  "version": "0.0.1",
```
Comment (jasongrout) [inline packages/nbconvert-css/package.json]: As a general rule, we start packages at 0.1.0 so that we can issue patch releases without breaking things.

Comment (SylvainCorlay) [reply packages/nbconvert-css/package.json]: Fixed!

**Comment by SylvainCorlay** [comment]
Ready!

**Comment by blink1073** [comment]
I am concerned that this ends up as part of the core build dependencies. I think we need to either only include core packages that have extension metadata or add another field for whether it should be included.  I'll open another issue to track that.  This looks good from my end as is, thanks!

**Comment by SylvainCorlay** [comment]
Thanks @blink1073!

**Comment by SylvainCorlay** [comment]
Am I allowed to push `@jupyterlab/nbconvert-css` to npm or do you guys have a process for that?

**Comment by blink1073** [comment]
Go for it, the version will get bumped when we do the final release anyway.  You already have the write permissions.

**Pull request merged**